init-plugin.sh

```diff
@@ -116,6 +116,10 @@ rm -f init-plugin.sh
 rm -f composer.lock
 rm -f package-lock.json
 
+# Setup PHPCS and PHPUnit config files
+mv phpcs.xml.dist phpcs.xml
```

Comment:
Could you explain this change.

Reply:
I did not realize that PHPCS and PHPUnit could use also the config file with the `.dist` extension. Which is cool. The only downside for me is that there is no code highlighting for these 2 files in a editor when using the `.dist` extension.

Reply:
We could rename the files to drop the `.dist` I suppose. Need to validate everything is still working as expected but I don't see an issue with that.